### ISSUE TITLE
MacOS Big Sur not working with Anoia (v2.11.1)

### ISSUE THREAD

**Issue by kaarta-dduggins** [CloudCompare/CloudCompare#1399]
It opens a white window and all the menu functions seem to work. It is not displaying any point clouds though.
![image](https://user-images.githubusercontent.com/22967660/107980607-94f42700-6f8e-11eb-97db-47089324d87a.png)

**Comment by kaarta-dduggins** [comment]
Appears to have been a video driver issue. I disconnected the Thunderbolt monitor and CC started normally on the lap top screen. I then plugged in the Thunderbolt monitor and CC kept working. Now it seems to come up normally even with the monitor connected. Below is the initial error thrown in the system log when it did not work initially. 

Feb 16 11:04:18 MacBook-Pro com.apple.xpc.launchd[1]: Coalition Cache Hit: app<application..18.21(501)> [19250]
Feb 16 11:04:18 MacBook-Pro CloudCompare[82993]: getattrlist failed for /System/Library/Extensions/AppleIntelKBLGraphicsGLDriver.bundle/Contents/MacOS/AppleIntelKBLGraphicsGLDriver: #2: No such file or directory
Feb 16 11:04:18 MacBook-Pro CloudCompare[82993]: getattrlist failed for /Library/GPUBundles/AMDRadeonX4000GLDriver.bundle/Contents/MacOS/ATIRadeonX4000SCLib.dylib: #2: No such file or directory
Feb 16 11:04:18 MacBook-Pro CloudCompare[82993]: getattrlist failed for /System/Library/Extensions/AMDRadeonX4000GLDriver.bundle/Contents/MacOS/ATIRadeonX4000SCLib.dylib: #2: No such file or directory
Feb 16 11:04:18 MacBook-Pro CloudCompare[82993]: getattrlist failed for /System/Library/Frameworks/OpenGL.framework/Resources//GLRendererFloat.bundle/GLRendererFloat: #2: No such file or directory

I think this issue can be closed.
